log.py

```diff
@@ -52,22 +54,40 @@ def format(self, record):
         return data
 
 class Logger(object):
-
-    DEBUG = "DEBUG"
-    INFO = "INFO"
-    WARN = "WARN"
-    ERROR = "ERROR"
```

Comment:
What happened to these variables?

Reply:
Specifically, with these gone, how does LOG_LEVEL_UI know what the options are?

Reply:
OK, I get it. These constants were redundant because they're also in LogConfiguration.